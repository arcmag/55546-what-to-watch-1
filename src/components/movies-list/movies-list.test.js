import React from "react";
import renderer from "react-test-renderer";
import MoviesList from "./movies-list.jsx";

const mockFilms = [
  {
    src: `test.jpg`,
    title: `Title film`
  },
  {
    src: `test.jpg`,
    title: `Title film`
  }
];

it(`Test MoviesList`, () => {
  const tree = renderer.create(<MoviesList films={mockFilms} />).toJSON();
  expect(tree).toMatchSnapshot();
});
