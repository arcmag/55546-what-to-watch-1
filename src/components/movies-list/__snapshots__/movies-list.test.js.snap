// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test MoviesList 1`] = `
<div
  className="catalog__movies-list"
>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <button
      className="small-movie-card__play-btn"
      onClick={[Function]}
      type="button"
    >
      Play
    </button>
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Macbeth"
        height="175"
        src="test.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Title film
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <button
      className="small-movie-card__play-btn"
      onClick={[Function]}
      type="button"
    >
      Play
    </button>
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Macbeth"
        height="175"
        src="test.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
      >
        Title film
      </a>
    </h3>
  </article>
</div>
`;
